cargo-dist/src/config.rs

```diff
@@ -87,6 +87,17 @@ pub struct DistMetadata {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub targets: Option<Vec<String>>,
 
+    /// The feature flags to be passed to the cargo build step.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub features: Option<Vec<String>>,
+
+    /// Enable default features for cargo build.
+    #[serde(
+        skip_serializing_if = "default_features::skip_condition",
+        default = "default_features::default"
+    )]
+    pub default_features: bool,
+
```

Comment:
You want this to be an `Option<bool>` because a package might want to force this to false even if the workspace sets it to true, so you need to distinguish "default" from "don't care".

Reply:
Ah, I went in with the assumption that in case it's don't care it defaults to true anyways (in `cargo build`), hence the weird default value :)

Reply:
also: good call on the polarity of default_features (my PR went for no_default_features because I was thinking about cli flags, but better to match Cargo.toml)